File: JavaSide/src/java/control/FeedbackManager.java
package control;

import entity.Feedback;
import entity.User;

/**
 * Created by jonah on 28/9/2017.
 */
public class FeedbackManager {

    DatabaseManager valid = new DatabaseManager();
    private Feedback[] FeedbackList = new Feedback[100];

    //assume that feedback cannot be deleted
    /**
     *
     * @param user
     * @param message
     * @return
     */
    public boolean addFeedback(User user, String message) {
        int i = 0;

        for (i = 0; i < FeedbackList.length; i++) {
            if (FeedbackList[i] == null) {
                FeedbackList[i] = new Feedback(user, message);
                valid.addFeedback(user.getUsername(), message);
                return true;
            }
        }
        return false; //fail to get feedback
    }

}

File: ASPWebsite/ASPWebsite/App_Code/Control/FeedbackManager.cs
using ASPWebsite.App_Code.Entity;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace ASPWebsite.App_Code.Control
{
    /// <summary>
    /// Feedback manager.
    /// Responsible for managing Feedback objects.
    /// </summary>
    public class FeedbackManager
    {
        private DatabaseManager dbManager = new DatabaseManager();

        /// <summary>
        /// Adds Feedback
        /// </summary>
        /// <returns><c>true</c>, if successful, <c>false</c> otherwise.</returns>
        /// <param name="username">User name.</param>
        /// <param name="message">Feedback message.</param>
        public Boolean addFeedback(string userName, string message)
        {
            Feedback toSubmit = new Feedback(userName, message, DateTime.Now);
            return dbManager.saveFeedback(toSubmit);
        }

        /// <summary>
        /// Get Feedbacks
        /// </summary>
        /// <returns>All feedbacks submitted.</returns>
        /// <param name="username">User name.</param>
        /// <param name="message">Feedback message.</param>
        public List<Feedback> getAllFeedback()
        {
            return dbManager.getFeedback();
        }
    }
}